src/PythonMigrationViewExtension/packages.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="DiffPlex" version="1.6.3" targetFramework="net48" />
+  <package id="DiffPlex.Wpf" version="1.1.1" targetFramework="net48" />
```

Comment:
One thing I forgot to mention is that new external libraries that we add references to should be mentioned in the license files. We have [LICENSE.txt](https://github.com/DynamoDS/Dynamo/blob/master/LICENSE.txt) at the root and [License.rtf](https://github.com/DynamoDS/Dynamo/blob/master/doc/distrib/License.rtf) under doc/distrib. Sorry in advance about the RTF editing, but we are using it because it displays nicely in the About window.

Reply:
@mmisol Do we need to update both and keep them in sync or?

Reply:
@QilongTang Yes, we should add entries for the libraries in both.

Reply:
@SHKnudsen Please address this.

Reply:
@mmisol added these dependencies to both Licences files